sstable/src/dictionary.rs

```diff
@@ -338,6 +338,44 @@ impl<TSSTable: SSTable> Dictionary<TSSTable> {
         Ok(true)
     }
 
+    /// Returns the terms for a _sorted_ list of term ordinals.
+    ///
+    /// Returns true if and only if all terms have been found.
+    pub fn ords_to_term_cb<F: FnMut(&[u8])>(
+        &self,
+        ord: impl Iterator<Item = TermOrdinal>,
+        mut cb: F,
+    ) -> io::Result<bool> {
+        let mut bytes = Vec::new();
+        let mut current_block_addr = self.sstable_index.get_block_with_ord(0);
+        let mut current_sstable_delta_reader =
+            self.sstable_delta_reader_block(current_block_addr.clone())?;
+        let mut current_ordinal = 0;
+        for ord in ord {
+            // check if block changed for new term_ord
+            let new_block_addr = self.sstable_index.get_block_with_ord(ord);
+            if new_block_addr != current_block_addr {
```

Comment:
So right now the facts that ord is sorted is only enforced by the comment.

We should either:

We can make sure this function works, but is slow, even for non sorted ord, as follows.

```suggestion
            if new_block_addr != current_block_addr || ord < current_ordinal {
```

Or we could add an assert

Reply:
I changed it to `sorted_ords_to_term_cb` + assert, since the performance hit is too big otherwise